lib/utils/socketChainHelper.js

```diff
@@ -53,6 +54,15 @@ const processServerResponse = toString => function processServerResponseHandler(
 		);
 	}
 
+	if (res.errorType === 'appRunning') {
```

Comment:
This is not what was planned actually. On line 14 we check if line is `queryFailed`, and if it is, and line is `testLine`, we throw AssertionError on line 48. The issue was that `application().hasExited()` chain fails not with `queryFailed_` type of error, but with `appRunning`. So basically we should simply add `appRunning` error type to test on line 14, so it is thrown as AssertionError.

Maybe it would be better to rename `const queryFailed` to `const isAssertionError` so it is more clear. 

There is a separate function for AssertionError message handling if we want some additional text for `application` and/or `until` chains.

Reply:
@AntonSBilan  please update according to @givehug comment.